libs/blocks/global-navigation/utilities/getUserEntitlements.js

```diff
@@ -24,35 +24,37 @@ const getQueryParameters = (params) => {
   return query.length ? `?${query.join('&')}` : '';
 };
 
-const emptyEntitlements = {
+const emptyEntitlements = () => ({
```

Comment:
I'm not sure I see a reason for this to be a function rather than a simple object. It doesn't return anything, it doesn't have arguments, it doesn't get updated, etc. So I'd vote for this to be a simple object. Test file also uses a simple object, so we should update here to be consistent.

Reply:
I agree with @overmyheadandbody - we should add some unit testing to cover this, I had the same question around using an object vs function.